libs/langchain/langchain/evaluation/comparison/eval_chain.py

```diff
@@ -27,6 +25,10 @@
 from langchain.evaluation.schema import LLMEvalChain, PairwiseStringEvaluator
 from langchain.schema import RUN_KEY
 
+if TYPE_CHECKING:
+    from langchain_community.chat_models.azure_openai import AzureChatOpenAI
```

Comment:
We'll need to do another pass on this in a bit for now just changing into a type checking dep